integration-tests/delete-partition.py

```diff
@@ -22,5 +22,6 @@
 
 print staging
 
+storage.calculate_actiongraph()
```

Comment:
Wouldn't it be better if `commit` detected that the action graph was not calculated yet and did it automatically? Or am I understanding the API wrong?